src/Symfony/Component/AssetMapper/Compiler/JavaScriptImportPathCompiler.php

```diff
@@ -28,16 +28,16 @@
 final class JavaScriptImportPathCompiler implements AssetCompilerInterface
 {
     /**
-     * @see https://regex101.com/r/1iBAIb/1
+     * @see https://regex101.com/r/1iBAIb/2
      */
     private const IMPORT_PATTERN = '/
         ^
             (?:\/\/.*)                     # Lines that start with comments
```

Comment:
```suggestion
            ^(?:\/\/.*)                     # Lines that start with comments
```

Reply:
dammit, I suck. Thanks for checking my work!